framework/helpers/BaseHtml.php

```diff
@@ -1955,10 +1955,8 @@ public static function renderTagAttributes($attributes)
                     foreach ($value as $n => $v) {
                         if (is_array($v)) {
                             $html .= " $name-$n='" . Json::htmlEncode($v) . "'";
-                        } else if (is_bool($v)) {
-                            if ($v) {
-                                $html .= " $name-$n";
-                            }
+                        } elseif (is_bool($v) && $v) {
+                            $html .= " $name-$n";
```

Comment:
This will break the `false` test, which should just prevent the attribute from being added at all (like other `false` attributes).

Reply:
Ah, right.